avm/src/vm/deploy.rs

```diff
@@ -88,7 +88,11 @@ async fn post(url: String, body: Value) -> Result<String, Box<dyn Error>> {
   let mut resp = client.request(req).await?;
   let data = hyper::body::to_bytes(resp.body_mut()).await?;
   let str = String::from_utf8(data.to_vec())?;
```

Comment:
I *just* noticed that you're getting a name clash since `str` is a primitive type. Maybe use `data_str`?

Reply:
Sure, will address that in the next PR which I already started working on.